utils/roles/remove_node/user_confirmation/tasks/main.yml

```diff
@@ -18,6 +18,19 @@
     msg: "{{ no_inventory_file_provided }}"
   when: ansible_inventory_sources[0] == default_inv_path
 
+- name: Default omnia_inventory_file is provided check
+  ansible.builtin.pause:
+    prompt: "{{ omnia_inventory_confirmation }}"
+  register: default_omnia_inventory
+  when: ansible_inventory_sources[0] == omnia_inventory_file
+
+- name: Fail if user does not confirm deletion
```

Comment:
even if this file is deleted, it will generate again as data is present in omniadb

Reply:
Ok thats good. but this deletion is for node